movies_import/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+# Create your models here.
+
+class MoviesImport(models.Model):
```

Comment:
Same here as for the previous import functionality. There is no need to store the zip we use to import files. Process it on the fly, when the file is received.